storage/preferences/src/main/java/de/danoeh/antennapod/storage/preferences/UserPreferences.java

```diff
@@ -867,4 +872,22 @@ public static void setSubscriptionsFilter(SubscriptionsFilter value) {
     public static boolean shouldShowSubscriptionTitle() {
         return prefs.getBoolean(PREF_SUBSCRIPTION_TITLE, false);
     }
+
+    public static void setAllEpisodeSortOrder(SortOrder s) {
```

Comment:
Not `setAllEpisodesSortOrder` (episode**S**)?

Reply:
good catch